### PR TITLE
Fix back-to-top button

### PR DESCRIPTION
**Pull request by tacigar**
before
current

after
fixed

### CODE REVIEW

File: zipkin-ui/js/component_ui/traces.js [hunk 1]
```diff
@@ -29,8 +28,6 @@ export default component(function traces() {
       this.timestamp = parseInt($this.data('timestamp'), 10);
       this.percentage = parseInt($this.data('servicePercentage'), 10);
     });
-    this.on(document, 'uiAddServiceNameFilter', this.addFilter);
```
Comment (tacigar) [inline zipkin-ui/js/component_ui/traces.js:32]: `addFilter` and `removeFilter` are removed when filter features are removed.
So this process does not finish successfully.
The process of this line seems to be called in https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/page/default.js#L89 , so later processes of this line will not be executed.
So https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/page/default.js#L92 is not called, and then https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/component_ui/backToTop.js#L11 is not called, too.

As a result, this callback https://github.com/openzipkin/zipkin/blob/2e32676489acd3c807cfcc57a6e0e456a00fd607/zipkin-ui/js/component_ui/backToTop.js#L13 is not set... 😢 

This is the cause of this bug.

File: zipkin-ui/js/component_ui/traces.js [hunk 2]
```diff
@@ -17,7 +17,6 @@ export default component(function traces() {
     if (this.sortFunctions.hasOwnProperty(data.order)) {
       this.$traces.sort(this.sortFunctions[data.order]);
       this.$node.html(this.$traces);
-      this.triggerUpdateTraces();
```
Comment (tacigar) [inline zipkin-ui/js/component_ui/traces.js:20]: There is not this method now

**Comment by codefromthecrypt** [comment]
you are a detective!

**Comment by codefromthecrypt** [comment]
thank you again for the close eyes and able hands @tacigar!

**Pull request merged**